controllers/update_pods.go

```diff
@@ -156,8 +156,8 @@ func (updatePods) reconcile(ctx context.Context, r *FoundationDBClusterReconcile
 	return deletePodsForUpdates(ctx, r, cluster, adminClient, updates, logger)
 }
 
-func getPodsToDelete(deletionMode fdbtypes.DeletionMode, updates map[string][]*corev1.Pod) (string, []*corev1.Pod, error) {
-	if deletionMode == fdbtypes.DeletionModeAll {
+func getPodsToDelete(deletionMode fdbtypes.PodUpdateMode, updates map[string][]*corev1.Pod) (string, []*corev1.Pod, error) {
```

Comment:
As a follow up we should move this method into the internal package and try to reuse the same mechanism that we are using for the process groups.